BridgeCareApp/VuejsApp/public/index.html

```diff
@@ -8,6 +8,8 @@
     <title>Bridge Care</title>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
Is this a CDN? If yes, shouldn't we install the packages instead.

Reply:
It is a CDN, and I would say yes, but we appear to already have a version of bootstrap installed. However, the vue query builder specified that it need a specific version of bootstrap. I'm not sure how installing multiple versions of bootstrap will work, or if it's even possible, but if we can do it that way I'm all for it.

Reply:
We can remove the installed version. I don't think that we are using bootstrap anywhere except in query builder.

Reply:
It seems the bootstrap cdn keeps the query builder visually correct whereas not having it makes the groupings less clear. I think we should keep the cdn as is.

Reply:
Okay, no problem with that